Android.bp

```diff
@@ -49,6 +49,8 @@ android_app {
     optimize: {
         enabled: false,
     },
+    sdk_version: "30",
```

Comment:
Could set to current, not sure if that's handled any differently though.

Reply:
I set it to `30` explicitly, so when you run this on Android 12 without making the required modifications, you'll notice here.

Reply:
Shouldn't it be the other way around for that.

Anyway, we shouldn't be making changes now for something next year, it's easy enough to know you're trying to build this without any changes in a new version.

And it might not even work, due to now using hidden-apis

"current" is handled differently from a specific version

Reply:
removed